### PR TITLE
[TRANSLATION] Improve Simplified Chinese (zh-CN) translation

### PR DESCRIPTION
**Pull request by RatinCN**
## Purpose

Improve Simplified Chinese (zh-CN) translation.

## Proposed changes

- Add and improve translation
- Fix mistakes

## TODO

- [x] More translations
- [x] Test

### CODE REVIEW

**Comment by RatinCN** [comment]
Cc @reactos/lang-chinese

**Comment by RatinCN** [comment]
New coming conflicts are resolved.

**Comment by binarymaster** [comment]
Re-pinging @reactos/lang-chinese

**Review by eason329 — COMMENTED** [review]
I saw that you have used quotation mark for Traditional Chinese environment (「 」). I think it would be better to follow the GB standard (“ ”).

**Comment by yangyangdaji** [comment]
> I saw that you have used quotation mark for Traditional Chinese environment (「 」). I think it would be better to follow the GB standard (“ ”).

This aligns with Windows XP

Windows XP

File: dll/shellext/netshell/lang/zh-CN.rc [hunk 1]
```diff
@@ -144,9 +144,9 @@ BEGIN
     IDS_SHV_COLUMN_TYPE "类型"
     IDS_SHV_COLUMN_STATE "状态"
     IDS_SHV_COLUMN_DEVNAME "设备名称"
-    IDS_SHV_COLUMN_PHONE "电话 # 或主机地址"
+    IDS_SHV_COLUMN_PHONE "电话号码或主机地址"
     IDS_SHV_COLUMN_OWNER "所有者"
-    IDS_TYPE_ETHERNET "局域网或高速互联网"
```
Comment (yangyangdaji) [inline dll/shellext/netshell/lang/zh-CN.rc:149]: revert it because it's better.

**Review by yangyangdaji — CHANGES_REQUESTED** [review]

**Comment by RatinCN** [comment]
> I saw that you have used quotation mark for Traditional Chinese environment (「 」). I think it would be better to follow the GB standard (“ ”).

Committed, and also changed existing `「开始」` to `“开始”`. `「」` used under NT5.2, but it's rarely used in simplified Chinese (GB standard). And now Win11 use `“开始”`, I vote to the latter too. Cc @eason329 @yangyangdaji .

File: dll/shellext/netshell/lang/zh-CN.rc (hunk 1 above)
Comment (RatinCN) [reply dll/shellext/netshell/lang/zh-CN.rc:149]: In Windows (from NT5.2 to the latest Win11), "LAN" and "Internet" are preserved as well-known vocabulary, not translated.

**Comment by kernelbin** [comment]
> itional Chinese environment (「 」). I think it would be better to follow the GB standard (“ ”).

I remember we reached an agreement on the use of parentheses a long time ago?

File: boot/bootdata/hivedef.inf [hunk 2]
```diff
@@ -4167,7 +4167,7 @@ POWERCFG_DESC_5="此方案能有效节省电池电力。"
 DESKTOP_SCHEME_0="ReactOS 标准"
 DESKTOP_SCHEME_1="红砖色"
 DESKTOP_SCHEME_2="茄子"
-DESKTOP_SCHEME_3="绿色橄榄"
+DESKTOP_SCHEME_3="淡绿色"
 DESKTOP_SCHEME_4="高对比度 1"
```
Comment (kernelbin) [inline boot/bootdata/hivedef.inf:4171]: is it better to translate as 橄榄绿?

**Comment by kernelbin** [comment]
Everything else looks good to me

File: dll/shellext/netshell/lang/zh-CN.rc (hunk 1 above)
Comment (yangyangdaji) [reply dll/shellext/netshell/lang/zh-CN.rc:149]: oh

**Review by yangyangdaji — APPROVED** [review]

**Review by eason329 — APPROVED** [review]

File: boot/bootdata/hivedef.inf (hunk 2 above)
Comment (RatinCN) [reply boot/bootdata/hivedef.inf:4171]: Yes it's better, committed.

**Review by yangyangdaji — APPROVED** [review]

File: base/applications/utilman/umandlg/lang/zh-CN.rc [hunk 3]
```diff
@@ -3,14 +3,15 @@
  * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
  * PURPOSE:     Chinese (Simplified) resource file
  * TRANSLATOR:  Copyright 2020 Zheng Jianping <robsean@126.com>
```
Comment (binarymaster) [inline base/applications/utilman/umandlg/lang/zh-CN.rc:5]:
```suggestion
 * TRANSLATORS: Copyright 2020 Zheng Jianping <robsean@126.com>
```

**Pull request merged**